sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ObjectExpressionsSuite.scala

```diff
@@ -71,6 +79,66 @@ class ObjectExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       mapEncoder.serializer.head, mapExpected, mapInputRow)
   }
 
+  test("SPARK-23582: StaticInvoke should support interpreted execution") {
```

Comment:
Will use `checkObjectExprEvaluation` in https://github.com/apache/spark/pull/20757/files